### PR TITLE
doc: add koa usage

### PR DESCRIPTION
**Pull request by MatanAvneri**
I added some more documentation on how to use it with Koa, and I fixed and existing format issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,11 @@ request({ method: 'POST', url: 'http://www.mocky.io/v2/5bd9984b2f00006d0006d1fd'
 ## Usage in koa
 This package supports koa server that uses [`koa-router`](https://www.npmjs.com/package/koa-router) and [`koa-bodyparser`](https://www.npmjs.com/package/koa-bodyparser)
 
+```js
+const apiMetrics = require('prometheus-api-metrics').koaMiddleware
```
Comment (kobik) [inline README.md]: maybe it's better to destructure?

`const { koaMiddleware } = require('prometheus-api-metrics')`

Comment (MatanAvneri) [reply README.md]: Yep, your'e right, it's shorter, changed

**Review by kobik — APPROVED** [review]

**Comment by kobik** [comment]
@MatanAvneri, can you sync with master?

**Comment by MatanAvneri** [comment]
> @MatanAvneri, can you sync with master?

Merged

**Pull request merged**